python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/slice_conditions.py

```diff
@@ -99,6 +99,22 @@ def compute_slice(self, context: SchedulingContext) -> AssetSlice:
         )
 
 
+@whitelist_for_serdes
+class NewlyUpdatedCondition(SliceSchedulingCondition):
```

Comment:
`UpdatedOnePreviousTick` or something? I think it is important that we understand that this is only true one tick/tick interval at a time.